### ISSUE TITLE
Create issues in issue tracking system to help guide people that are interested in contributing

### ISSUE THREAD

**Issue by jonathanrfisher1** [BootsOnTheGround-3Caretakers/three-caretakers-v001#2]
It's difficult to come into a project and just figure out how you can help by looking at the repo.

This project could use some tickets created by the maintainer to help people understand where to get started and where help is needed.  There are other standards similar to README files that could be created as well like a CONTRIBUTING.md file for example.  A ticket could be created for making these things =)

**Comment by chonghorizons** [comment]
Yeah, I agree. We are onboarding people manually since the codebase is changing fairly quickly. I'm not used to setting up big open source projects so I'm not familiar with the standards.

Today I'm working on the toddlerApi branch to redo the schema.
